### PR TITLE
Fix/improve admin workflow

### PR DESCRIPTION
**Pull request by mspector**

### CODE REVIEW

File: admin/superuser-init.py [hunk 1]
```diff
@@ -60,95 +60,96 @@ def get_input_or_data(name):
...
-print "writing %s/process/commentProcess.js" % directory
-with open(directory + '/process/commentProcess.js', 'w') as new_commentprocess, open(utils.get_path('./conference-template/process/commentProcess.template', __file__)) as template_commentprocess:
-    templatestring = template_commentprocess.read().replace('<<PHRASE>>', "\"%s\"" % conference_phrase)
-    new_commentprocess.write(templatestring)
-
-print "writing %s/params.json" % directory
-with open(directory + '/params.json', 'w') as paramsfile:
-    json.dump(data, paramsfile, indent=4)
+
+if args.overwrite or not os.path.exists(directory):
```
Comment (melisabok) [inline admin/superuser-init.py]: where do you create the root group now? I don't see the call

Comment (mspector) [inline admin/superuser-init.py]: it gets created on line 176

Comment (melisabok) [inline admin/superuser-init.py]: I'm running the superuser-init for the semantic web conference and I get an error trying to create the admin user:
screen shot 2017-06-16 at 10 09 44 am


I'm using the suggested admin id

Comment (mspector) [inline admin/superuser-init.py]: sorry, the "suggestion" isn't a "default" - you would still have to enter the username you want. I can change this.

Comment (melisabok) [inline admin/superuser-init.py]: Oops, I'm sorry.

Now it is working fine. I could create the semantic web conference.

**Pull request merged**